### PR TITLE
Fix wrong import

### PR DESCRIPTION
**Pull request by 2O4**
Fix wrong import from `url` (unused in example) to `path`.

Fixes #

Just a quick fix.

### CODE REVIEW

File: docs/tutorial/tutorial_02.rst [hunk 1]
```diff
@@ -34,7 +34,7 @@ URL this view will respond to:
 
 .. code-block:: python
 
-    from django.conf.urls import url, include
```
Comment (auvipy) [inline docs/tutorial/tutorial_02.rst:37]: it is right import, to import path you need to import from: django.urls

**Review by auvipy — DISMISSED** [review]

**Comment by n2ygk** [comment]
@auvipy You are merging without confirming the PR template was filled out (again).  Please follow the process.

**Comment by amureki** [comment]
@n2ygk my bad, apologies! I did merge it and missed the point that we do not have a proper PR description followed by a template.

**Comment by auvipy** [comment]
> @auvipy You are merging without confirming the PR template was filled out (again). Please follow the process.

I did not  merge it

**Comment by n2ygk** [comment]
Apologies. I misread the log.

**Pull request merged**